docs/output.md

```diff
@@ -2,66 +2,136 @@
+    - [MultiQC](#multiqc) - Aggregate report describing results
+    - [BBduk](#bbduk) - Filter out sequences from samples that matches sequences in a user-provided fasta file (optional)
+  - [Filtering genomes](#filter-genomes-step) - Generate a list of genomes that will be used for the mapping
+    - [Sourmash](#sourmash) - Output from Sourmash filtering of genomes.
+  - [ORF Caller step](#orf-caller-step) - Identify protein-coding genes (ORFs) with an ORF caller
+    - [Prokka](#prokka) - Output from Prokka (optional)
+  - [Mapping step](#mapping-reads-to-genomes) - Predict the function and the taxonomy of ORFs
+    - [BBmap](#bbmap) - Output from BBmap
+    - [Featurecounts](#featurecounts) - Output from FeatureCounts
+- [Custom magmap output](#magmap-output)
+  - [Summary tables folder](#summary-tables) - Tab separated tables ready for further analysis in tools like R and Python
+  - [Pipeline information](#pipeline-information) - Report metrics generated during the workflow execution
+
+## Module output
 
-* [FastQC](#fastqc) - Raw read QC
-* [MultiQC](#multiqc) - Aggregate report describing results and QC from the whole pipeline
-* [Pipeline information](#pipeline-information) - Report metrics generated during the workflow execution
+### Preprocessing
 
-### FastQC
+#### FastQC
+
+[FastQC](http://www.bioinformatics.babraham.ac.uk/projects/fastqc/) gives general quality metrics about your sequenced reads. It provides information about the quality score distribution across your reads, per base sequence content (%A/T/G/C), adapter contamination and overrepresented sequences. For further reading and documentation see the [FastQC help pages](http://www.bioinformatics.babraham.ac.uk/projects/fastqc/Help/). FastQC is run as part of Trim galore! therefore its output can be found in Trimgalore's folder.
 
 <details markdown="1">
 <summary>Output files</summary>
 
-* `fastqc/`
-    * `*_fastqc.html`: FastQC report containing quality metrics.
-    * `*_fastqc.zip`: Zip archive containing the FastQC report, tab-delimited data file and plot images.
+- `trimgalore/fastqc/`
+  - `*_fastqc.html`: FastQC report containing quality metrics for your untrimmed raw fastq files.
 
 </details>
 
-[FastQC](http://www.bioinformatics.babraham.ac.uk/projects/fastqc/) gives general quality metrics about your sequenced reads. It provides information about the quality score distribution across your reads, per base sequence content (%A/T/G/C), adapter contamination and overrepresented sequences. For further reading and documentation see the [FastQC help pages](http://www.bioinformatics.babraham.ac.uk/projects/fastqc/Help/).
+#### Trim galore!
+
+[Trimgalore](https://github.com/FelixKrueger/TrimGalore) is trimming primer sequences from sequencing reads. Primer sequences are non-biological sequences that often introduce point mutations that do not reflect sample sequences. This is especially true for degenerated PCR primers.
```

Comment:
```suggestion
[Trim galore!](https://github.com/FelixKrueger/TrimGalore) is trimming primer sequences from sequencing reads. Primer sequences are non-biological sequences that often introduce point mutations that do not reflect sample sequences. This is especially true for degenerated PCR primers.
```

Please check how the authors format the name, and apply this consistently everywhere (for all tools!)